pallets/bootstrap/src/lib.rs

```diff
@@ -134,9 +134,12 @@ use frame_support::{
 	transactional, PalletId,
 };
 use frame_system::{ensure_root, ensure_signed, pallet_prelude::OriginFor};
+
 use mangata_support::traits::{
-	AssetRegistryApi, GetMaintenanceStatusTrait, PoolCreateApi, RewardsApi,
+	AssetRegistryApi, GetMaintenanceStatusTrait, PoolCreateApi, ProofOfStakeRewardsApi,
 };
+use mangata_types::multipurpose_liquidity::ActivateKind;
+
 use mangata_types::{Balance, TokenId};
 use orml_tokens::{MultiTokenCurrencyExtended, MultiTokenReservableCurrency};
 use pallet_vesting_mangata::MultiTokenVestingLocks;
```

Comment:
checked the code and extrinsic are missing documentation. Please add some high-level docs as we have in bootstrap, ...